chainerx_cc/chainerx/cuda/cuda_conv_test.cc

```diff
@@ -62,9 +62,9 @@ TEST(CudaConvTest, FwdAlgoCache) {
         bool cover_all = false;
 
         EXPECT_EQ(size_t{0}, cuda_internal::CudaConvTest::GetFwdAlgoCacheMapSize(cuda_conv));
-        cuda_conv.Conv(device, x, w, b, stride, pad, cover_all, x.dtype());
+        cuda_conv.Conv(device, x, w, b, stride, pad, {1, 1}, cover_all, x.dtype());
```

Comment:
A nit but how about not inlining "dilate" to be consistent with `stride`, `pad`, etc. and ditto for remaining tests. It's a detail so please ignore it if you disagree.